Update README for non-eu cypress runs (#3360)
bbc/simorgh: README.md

CYPRESS_APP_ENV=test CYPRESS_UK=true npm run cypress:interactive
```

#### Running e2e outside EU

> **This affects developers based out of the EU (but may affect you if you're using a VPN routing through a country in the EU)**

Reviewer: ```suggestion
**This affects developers based out of the EU (but may affect you if you're using a VPN routing through a country not in the EU)**
```

> **This affects developers based out off the EU (but may affect you if you're using a VPM routing through a country in the EU)**

Reviewer 2: VPN? Otherwise looks good :)

Reviewer 3: That fix has been made @oluoluoxenfree

> **This affects developers based out off the EU (but may affect you if you're using a VPN routing through a country in the EU)**

Reviewer 3: ```suggestion
**This affects developers based outside off the EU (but may affect you if you're using a VPN routing through a country in the EU)**
```

> **This affects developers based outside off the EU (but may affect you if you're using a VPN routing through a country in the EU)**

Reviewer 4: Please change ‘off’ to ‘of’

Or maybe just delete ‘off’

> Running Cypress tests outside the EU will not show the EU consent banners, and this may cause some tests to fail. Set `CYPRESS_SKIP_EU=true` to prevent these tests from failing when outside the EU.

Reviewer 5: ```suggestion
Running Cypress tests outside the EU will not show the EU consent banners on AMP, and this may cause some tests to fail. Set `CYPRESS_SKIP_EU=true` to prevent these tests from running when outside the EU.
```

Reviewer 6: > Set `CYPRESS_SKIP_EU=true` to prevent these tests from running when outside the EU.

that doesn't sound right 😖
